### DISCUSSION TITLE
"Generic" driver and "failed_when_contains" list

### DISCUSSION THREAD

**Discussion by ikievite** [scrapli/scrapli_community#150] [Q&A] [answered]
Hi
Could you tell if `failed_when_contains` list can be in `SCRAPLI_PLATFORM` dictionary with `Generic` `driver?`
Or `failed_when_contains` can be used only with `Network` driver?

**Comment by carlmontanari** [comment, marked as answer]
hey @ikievite this only exists with "network" drivers, but it really is just checking if some substrings are in the greater output, so you can easily do this yourself.
